uni/assets/text/TermsAndConditions.md

```diff
@@ -4,50 +4,144 @@
+omissions by you, including any damages of any kind incurred as a result of such acts or omissions.
+Any interaction with your data from Universidade do Porto's platform is made directly through it,
+and is not collected by us in any way.
+If you choose to select the option to keep you logged in, you understand that your sign-in details
+may be stored locally, but not in any way shared with us. These details are stored in the
+application data and we hold no responsibility for ensuring their concealment.
+\
+\
+**Ownership of Content:**
+
+We do not own any data, information or material ("Content") that you submit in the Mobile
+Application in the course of using the Service. You shall have sole responsibility for the accuracy,
+quality, integrity, legality, reliability, appropriateness, and intellectual property ownership or
+right to use of all submitted Content.
+\
+\
+**Monitoring and Reviewing:**
+
+We may, at our discretion but with no obligation, monitor and review Content in the Mobile
+Application submitted or created using our Services by you.
+\
+\
+**License and Usage Restrictions:**
+
+You grant us permission to access, copy, distribute, store, transmit, reformat, display and perform
+the Content of your user account solely as required for the purpose of providing the Services to
+you.
+Unless specifically permitted by you, your use of the Mobile Application does not grant us the
+license to link, reproduce, adapt, modify, publish or distribute the Content created by you or
+stored in your user account, for evaluating the effectiveness of existing product features, planning
+new features, or measuring size and characteristics about specific end-users, or devices with a data
+broker, for commercial, marketing or any similar purpose. Unless specifically permitted by you, your
+use of the Mobile Application does not grant us the
+license to use analytics tools, advertising networks, third-party SDKs, or other external vendors
+whose code we have added to the Uni Mobile Application, to collect information such as user ID,
+device ID, or profiles with Third-Party Data for the purpose of accessing necessary data to
+implement
+the Services we provide. We do not have the license to sell or share sensitive data accessed trough
+API' permissions with the
+finality of facilitating sales.
+\
+\
+**Data Collection and Privacy:**
+
+Personal data of users is not collected for any purpose other than providing the Services to
+Personal data of users is not collected for any other purpose, unless providing the Services to you,
+except in certain circumstances described
+in [Play Store’ Privacy policy](https://support.google.com/googleplay/android-developer/answer/10144311?visit_id=638365485539535125-3072678242&rd=1)
+and [App Stores privacy policy](https://developer.apple.com/app-store/app-privacy-details/#user-tracking).
+\
+\
+**All user privacy and data use is in compliance with Play Store and App Store Policies mentioned
+above.**
```

Comment:
This seems a bit out of place to me. What about mentioning this under *Monitoring and Reviewing*?